MichaelAng/dnd-char-gen
# Feature/38/add routing

# Issue

PR #38 
# Description

Add basic routing to the application
# Changes
- Created a route and hooked up spell list 
# Steps to test
1. run `docker-compose build && docker-compose up -d`
2. [ ] Verify it says hello world 
3. [ ] Verify it automatically navigates to spells
# Help

[wiki](https://github.com/MichaelAng/dnd-char-gen/wiki)
